src/command_modules/azure-cli-storage/HISTORY.rst

```diff
@@ -4,7 +4,8 @@ Release History
 ===============
 2.2.2
 +++++
-* Minor fixes
+* `--auth-mode login` parameter allows use of user's login credentials for blob and queue authorization.
```

Comment:
Rebase on oauth login commit should remove this from new addition?

Reply:
I forgot to add it with the oauth PR. I made a separate PR(https://github.com/Azure/azure-cli/pull/7229) to add the history as the service for this feature may not be ready for Ignite and this may not get merged.